action.yaml

```
@@ -60,6 +60,10 @@ inputs:
     required: false
     default: true
     description: "Copy pools from the original Deployment to the new deployment preview."
+  cli-version:
+    required: false
+    default: ""
+    descriptions: "The desired Astro CLI version to use"
```

```suggestion
    description: "The desired Astro CLI version to use"
```

There's only `description` in https://docs.github.com/en/enterprise-cloud@latest/actions/creating-actions/metadata-syntax-for-github-actions#inputs.